frontend/src/container/LogDetailedView/ContextView/ContextView.styles.scss

```diff
@@ -1,3 +1,28 @@
 .log-context-container {
     border: 1px solid var(--bg-slate-400);
-}
\ No newline at end of file
+    height: auto;
+    max-height: 80vh;
+    position: relative;
+    overflow: scroll;
+    overflow-x: hidden;
+
+    &::-webkit-scrollbar {
+        width: 0.3rem;
+    }
+
+    &::-webkit-scrollbar-track {
+        background: transparent; 
+    }
+    
+    &::-webkit-scrollbar-thumb {
+        background: var(--bg-slate-300);
+    }
+    
+    &::-webkit-scrollbar-thumb:hover {
+        background: var(--bg-slate-200);
+    }
+}
+
+.isEditable {
+    height: 73vh; // When edit mode is enabled, reducing the size of log-context-container to avoid unnecessary scroll bars.
```

Comment:
Let's move towards responsive design and avoid fixed heights and widths

Reply:
+1 , never set heights. only play with widths

Reply:
Removed it. Now for height we are using `flex: 1` to take the remaining available height.